src/internal/components/InternalLayoutEditor.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  Puck,
+  Data,
+  Config,
+  type History,
+  InitialHistory,
+} from "@measured/puck";
+import React from "react";
+import { useState, useRef, useCallback } from "react";
+import { getVisualConfigLocalStorageKey } from "../utils/localStorageHelper.ts";
+import { TemplateMetadata } from "../types/templateMetadata.ts";
+import { EntityFieldProvider } from "../../components/EntityField.tsx";
+import { LayoutSaveState } from "../types/saveState.ts";
+import { LayoutHeader } from "../puck/components/LayoutHeader.tsx";
+import { DevLogger } from "../../utils/devLogger.ts";
+
+interface InternalLayoutEditorProps {
```

Comment:
Not you, but can we change these to types instead of interfaces? 99% of the time types should be preferred. Same in other places like InternalThemeEditor.